app/views/account/pages/show.html.slim

```diff
@@ -1,7 +1,7 @@
 .container
   .row
     .col-md-12
-      = link_to  "Back to all pages", account_pages_path, class: 'btn btn-primary'
+      = link_to  "Back to all pages", account_company_pages_path, class: 'btn btn-primary'
```

Comment:
shouldn't this take a `company` arg?

Reply:
yes, it's a bug

Also .... it your code? And /account/users too. I don't understand why you have in PR code from other people? @Kmalynss  this is because of rebase? Maybe let's use merge? github could merge all commits together before merge?

Reply:
@igorkasyanchuk most likely it is due to mistakes during rebase.

Reply:
@igorkasyanchuk also this might be appearing due to conflict of this branch with master.